pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -480,6 +488,24 @@ def make_package(args):
             base_service_class=base_service_class,
         )
 
+    for spec in args.workers:
+        spec = spec.split(':')
+        name = spec[0]
+        entrypoint = spec[1]
+
+        worker_target_path = \
+            'src/main/java/{}/{}Worker.java'.format(
+                args.package.replace(".", "/"),
+                name.capitalize()
+            )
```

Comment:
Minor: we could make a one-liner helper function/method that does that, so we have a single source of truth.
```python
def get_target_path(package, name):
    return 'src/main/java/{}/Service{}.java'.format(package.replace(".", "/"), name.capitalize())
```